s3_storage_provider.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, hs, config):
 
         if "secret_access_key" in config:
             self.api_kwargs["aws_secret_access_key"] = config["secret_access_key"]
+        
+        if "verify" in config:
+            self.api_kwargs["verify"]=config["verify"]
```

Comment:
For reference, [this is the set of allowed properties](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session.client) passed to the client.

Reply:
```suggestion
        if "verify" in config:
            self.api_kwargs["verify"] = config["verify"]
```